connector/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala

```diff
@@ -70,7 +70,7 @@ class KafkaTestUtils(
   private val JAVA_AUTH_CONFIG = "java.security.auth.login.config"
 
   private val localHostNameForURI = Utils.localHostNameForURI()
-  logInfo(s"Local host name is $localHostNameForURI")
+  logInfo(log"Local host name is ${MDC(LogKey.URI, localHostNameForURI)}")
```

Comment:
Because the LogKey `URI` is duplicated with the J`ava class`, we will write it directly as `LogKey.XXX`
At the same time, we will delete the import 
```
import org.apache.spark.internal.LogKey.ERROR
```